test/HandleHumanLocalMUnitTests.hs

```diff
@@ -51,6 +52,12 @@ handleHumanLocalMUnitTests = testGroup "handleHumanLocalMUnitTests"
                    in chooseItemProjectHuman testActorId triggerItems
       result <- executorCli testFn testCliStateWithItem
       showFailError (fromJust (fst result)) @?= "*aiming obstructed by terrain*"
+  , testCase "tutorialHints-msg-in-history" $ do
+      let testFn = tutorialHintMsgAdd CannotHarmYouInMelee
+      result <- executorCli testFn testCliStateWithHint
+      let history =  shistory <$> (cliSession . snd)  result
+          hasTutorialHints= anyInReport ( ==  MsgClassShowAndSave MsgTutorialHint) . newReport <$> history
```

Comment:
How about the test I proposed before? Don't check the message class, but check if there is a message that is equal to `renderTutorialHints CannotHarmYouInMelee` (perhaps with some fuzz)?